File: card-core/src/main/java/com/jean202/cardmizer/core/port/in/GetCardPerformancePolicyUseCase.java
package com.jean202.cardmizer.core.port.in;

import com.jean202.cardmizer.core.domain.CardId;
import com.jean202.cardmizer.core.domain.CardPerformancePolicy;

public interface GetCardPerformancePolicyUseCase {
    CardPerformancePolicy get(CardId cardId);
}

File: card-core/src/main/kotlin/com/jean202/cardmizer/core/port/in/GetCardPerformancePolicyUseCase.kt
package com.jean202.cardmizer.core.port.`in`

import com.jean202.cardmizer.core.domain.CardId
import com.jean202.cardmizer.core.domain.CardPerformancePolicy

fun interface GetCardPerformancePolicyUseCase {
    fun get(cardId: CardId): CardPerformancePolicy
}
